### PR TITLE
[TW-1704] Doc add the ability to compare performance runs

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added renaming performance runs
- Add comparing performance runs
- Updates screenshots

View the beta docs:
- [Rename a performance run](https://tw-1704-compare-performance-runs.learning.postman-beta.com/docs/collections/performance-testing/performance-test-configuration/#rename-a-performance-run)
- [Compare two performance runs](https://tw-1704-compare-performance-runs.learning.postman-beta.com/docs/collections/performance-testing/performance-test-metrics/#compare-two-performance-runs)

*DONE: Need to update compare runs screenshot after switch icon is removed*

### CODE REVIEW

File: src/pages/docs/collections/performance-testing/performance-test-metrics.md [hunk 1]
```diff
@@ -76,3 +79,27 @@ To change the graph back to the default view, select **Reset**.
 To view details for a section of the graph, select a point on the graph. The table below the graph shows total requests, throughput, response time, and error metrics for each request during the selected time period. To change the graph back to the default time period, select **Clear Filter**.
 
 <img alt="Viewing a time period" src="https://assets.postman.com/postman-docs/v10/performance-test-zoom-v10-15.jpg" width="620px"/>
+
+## Compare two performance runs
+
+APIs in service tend change with the addition of new features, improvements, and bug fixes. Over time, these changes can have an impact on the API's performance.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/performance-testing/performance-test-metrics.md:85]:
```suggestion
APIs in service tend to change with the addition of new features, improvements, and bug fixes. Over time, these changes can have an impact on the API's performance.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/performance-testing/performance-test-metrics.md [hunk 2]
```diff
@@ -76,3 +79,27 @@ To change the graph back to the default view, select **Reset**.
...
+
+To compare two performance runs, do the following:
+
+1. Open the [past performance run](/docs/collections/performance-testing/performance-test-configuration/#view-past-performance-runs) you want to compare other runs to. This will be the reference run.
+1. Select <img alt="Full screen icon" src="https://assets.postman.com/postman-docs/icons/icon-compare.jpg#icon" width="16px"/> **Compare runs** to enter comparison mode.
+1. Choose a performance run to compare to the reference run. Select a recent run from the list, or search for a run by name.
+1. Use the performance view to compare the selected runs:
+
+    * The graph shows performance data for the reference run (solid lines) and the comparison run (dotted lines). Hover over a point on the graph to view more details.
+    * The table below the graph shows how values in the comparison run differ from the reference run. A value in green (<span style="color:green;">&#9660;</span>) indicates better performance and a value in red (<span style="color:red;">&#9650;</span>) indicates worse performance.
```
Comment (postfrzero) [inline src/pages/docs/collections/performance-testing/performance-test-metrics.md:99]: Just a note: Our usage of red/green is slightly more nuanced, though I'm not sure if we should explicitly communicate that to users in the documentation. We only use red/green to communicate "good" or "bad" changes if we're sure that the meaning of the change is universally consistent. E.g. an increase in response times is universally bad, but a change in the total number of requests sent during a test could be good or bad depending on the test configuration.

**Review by postfrzero — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/performance-testing/performance-test-metrics.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/collections/performance-testing/performance-test-metrics.md:99]: @postfrzero I added one more line: 
> Values in gray indicate differences that are not necessarily better or worse.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**